hw/ip/otbn/data/otbn.hjson

```diff
@@ -102,30 +102,76 @@
         }
       ]
     } // register : status
-    { name: "ERR_CODE",
-      desc: "Error Code",
+    { name: "ERR_BITS",
+      desc: '''
+        Error bitfield, reads as non-zero if an error was seen during OTBN
+        operation
+      ''',
       swaccess: "ro",
       hwaccess: "hwo",
       fields: [
-        { bits: "31:0",
-          name: "err_code",
+        { bits: "0",
+          name: "bad_data_addr"
           desc: '''
-            The error cause if an error occurred.
-
-            This is set to zero (ErrCodeNoError) when OTBN successfully
-            completes an operation or a positive value when OTBN stops on an
-            error.
-
-            While OTBN is running (!!CMD.busy set), this has no meaning. When
-            OTBN stops this will be set to indicate the outcome. It is
-            guaranteed not to change until OTBN is run again (with !!CMD.start).
-
-            See <a href='#error-conditions'>Error Conditions</a> for
-            details on the possible values.
+            A DMEM read or write occurred with an out of bounds or unaligned
+            address.
           '''
         }
+        { bits: "1",
+          name: "bad_insn_addr"
+          desc: '''
+            A DMEM read or write occurred with an out of bounds or unaligned
+            address.
+          '''
+        }
+        { bits: "2",
+          name: "call_stack"
+          desc: '''
+            A instruction tried to pop from an empty call stack or push to a
+            full call stack.
+          '''
+        }
+        { bits: "3",
+          name: "illegal_insn"
+          desc: '''
+            <ul>
+              <li>An instruction being excuted had an invalid encoding.</li>
+              <li>An access occurred for an invalid CSR or WSR.</li>
+              <li>
+                A CSR or WSR access occurred that is not permitted (e.g. writing
+                to a read-only CSR or WSR).
+              </li>
+            </ul>
+          '''
+        }
+        { bits: "4",
+          name: "loop"
+          desc: '''
+            <ul>
+              <li>A loop was started with a 0 iteration count.</li>
+              <li>
+                The final instruction of a loop was a branch or another loop.
+              </li>
+              <li>
+                The loop stack would overflow (loop nesting level too deep).
```

Comment:
Maybe this would be better if it used the same formulation as you do with the call stack above?